src/elm/Data/Multipliers/Resource.elm

```diff
@@ -62,3 +63,17 @@ currentPrice model level =
 config : Config.ResourceMultiplier
 config =
     Config.resourceMultiplierConfig
+
+
+setFromList : List ( Config.Level, Int ) -> Model -> Model
+setFromList list initialModel =
+    List.foldl
+        (\( level, int ) dict ->
+            AllDict.update level
```

Comment:
Are we updating an existing value or overwriting it? If we're overwriting we can use `AllDict.insert` and skip the `Maybe.map` stuff.

Reply:
we are updating an existing value here.